Composer/packages/client/src/pages/language-understanding/LUPage.tsx

```diff
@@ -26,7 +26,7 @@ const LUPage: React.FC<RouteComponentProps<{
 
   const { setCurrentPageMode } = useRecoilValue(dispatcherState);
   useEffect(() => {
-    setCurrentPageMode('lu');
+    setCurrentPageMode('language-understanding');
```

Comment:
Shouldn't this get removed?

Reply:
Thanks for catching this - yes, it should.